src/routes/(admin)/admin/e-sim/[slug]/+page.server.js

```
@@ -1,12 +1,12 @@
 import { api } from "$lib/helpers";
 import { arktype } from "sveltekit-superforms/adapters";
-import { eSimDefaults, eSimSchema, brandSchema, brandDefaults, brandEditSchema, brandEditDefault } from "$lib/schemas";
+import { eSimDefaults, eSimSchema } from "$lib/schemas";
 import { message, superValidate, fail, setError } from "sveltekit-superforms";
 
 /** @type {import('./$types').PageServerLoad} */
 export async function load(event) {
-  // const form = await superValidate(arktype(giftCardSchema, { defaults: giftCardDefaults }));
-  const brandForm = await superValidate(arktype(brandSchema, { defaults: brandDefaults }));
+  const form = await superValidate(productData.data, arktype(eSimSchema, { defaults: eSimDefaults }));
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)